### ISSUE TITLE
Idea: More tabs on profiles

### ISSUE THREAD

**Issue by trwnh** [pixelfed/pixelfed#307]
Per IRC: we have two tabs on profiles currently:
- Posts: this shows all posts made by an account
- Saved: this shows posts you have bookmarked, and is only visible to you

Proposals for additional tabs:
- Collections: see #270
- Boosts: either make a view only for boosts/reshares, or alternatively change "Posts" to "{user}'s Posts" / "Posts and shares"
- Comments: view comments you've made on other people's posts. I would personally propose making this a private tab like Saved.
- A more general "updates" timeline for text-only posts, similar to iTunes Ping

```
[22:08:59]  <trwnh> dansup: Summary posts sounds interesting... as for pinned
                    posts though, I think it would be better to implement a
                    "highlight reel" option. People would highlight a post and
                    it would appear as a separate tab on their profile
[22:09:08]  <trwnh> Either that or have dynamic collections ;)
[22:09:24]  <trwnh> The abstraction from one to the other is probably not that
                    hard
[22:09:27] <dansup> trwnh: interesting!
[22:10:13]  <trwnh> dansup: The way I envision it right now is that profiles
                    need a third tab, and if that tab is Collections, then there
                    could be a Highlights collection
[22:10:55]  <trwnh> I think I might make a mockup of that later
[22:13:15] <dansup> trwnh: I think tabs are ok if we implement it right
[22:13:28] <dansup> collections should be called albums and show posts with more
                    than 1 media
[22:13:34]  <trwnh> dansup: it just feels unbalanced with only two tabs
[22:13:44]  <trwnh> especially if the profile is 3-column thumbnails
[22:13:56] <dansup> yeah, but think about IG
[22:14:12]  <trwnh> ehh
[22:14:25]  <trwnh> consideration to IG should be given but not copied 100%
[22:14:25]    <lcp> eh, class I made for it will adapt to any amount of tabs
[22:14:26] <dansup> if this were twitter/mastodon it would have 3 tabs. Posts, 
                    Posts w/ replies, Media
[22:14:49]  <trwnh> I don't know if it would be useful to show posts with
                    multiple media in its own tab
[22:14:50]    <lcp> you can make 20 tabs >D
[22:14:57]  <trwnh> That doesn't feel like it warrants its own tab
[22:15:27]  <trwnh> Arbitrary collections, though? A highlight reel? That would 
                    be fundamentally different than posting things
[22:15:34]    <lcp> it would probably be: posts, comments, boosts
[22:15:46]    <lcp> so two more things than primary view
[22:16:11]  <trwnh> But yeah, maybe a comments tab? Although I'd side against 
                    that, thinking about it. Probably not a good idea to let 
                    people find every single comment you've ever made?
[22:16:23]    <lcp> what about boosts
[22:17:01]  <trwnh> Basically, best candidates for new tabs would be collections
                    or boosts, as lcp brings up
[22:17:26]  <trwnh> Or perhaps "All posts" / "Original posts" / "Collections"
[22:17:38]  <trwnh> because I just remembered "saved" is private and not shown 
                    to everyone
[22:18:08]  <trwnh> Just, fundamentally, you want a way to sort through your own
                    posts, somehow or another
[22:19:58]    <lcp> well, I would imagine own posts would still be primary 
                    profile view
[22:20:57]  <trwnh> yeah
[22:26:53] <dansup> trwnh & lcp: can one of you make an issue for profile tabs?
[22:27:52]  <trwnh> yeah, i'll do it
```

**Comment by dansup** [comment]
Fixed in #1511

**Comment by trwnh** [comment]
#1511 adds Collections but doesn't really do anything for boost / comment / status tabs (if we're still doing those.) On the other hand, the Collections issue is #270 so that's probably what #1511 should close
